services/worker-manager/src/api.js

```diff
@@ -365,6 +365,100 @@ builder.declare({
   return res.reply(data.serializable());
 });
 
+builder.declare({
+  method: 'put',
+  route: '/workers/:workerPoolId:/:workerGroup/:workerId',
+  name: 'createWorker',
+  title: 'Create a Worker',
+  stability: APIBuilder.stability.experimental,
+  input: 'create-worker-request.yml',
+  output: 'worker-full.yml',
+  // note that this pattern relies on workerGroup and workerId not containing `/`
```

Comment:
what happens if they do?

Reply:
The schema prohibits it, so it will be an error -- the comment is just to suggest that if we change the schema, we're gonna have issues here.